spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -442,6 +436,14 @@ def main(argv: Sequence[str]):
         print_fail()
         print(err)
 
+    # check if data folder contains the required default depedencies
+    for dataset_name in default_datasets() + [_which_sct_binaries()]:
+        print_line(f"Check data dependency '{dataset_name}'")
+        if is_installed(dataset_name):
+            print_ok()
+        else:
+            print_fail(f" Run 'sct_download_data -d {dataset_name}' to reinstall")
```

Comment:
New check added!

Success logs:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/1d5c7a57-5e21-4709-875e-2b7f25bbe5af)

I just need to fixup my batch script syntax to actually install the dependencies on Windows :sweat_smile: 

Failing logs:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/c1168a9d-97fb-48e2-a79d-db5e3c422b03)